pkg/pillar/zedcloud/authen.go

```diff
@@ -422,36 +440,43 @@ func checkMimeProtoType(r *http.Response) bool {
 	return mimeType == ctTypeProtoStr
 }
 
-// VerifySigningCertChain - verify signing certificate chain from controller
-func VerifySigningCertChain(ctx *ZedCloudContext, content []byte) ([]byte, error) {
+// VerifyZControllerCert - unmarshal and verify the content of ZControllerCert
+// receiver from the controller.
+// Returns content of the signing certificate and the verification error/nil value.
+func VerifyZControllerCert(log *base.LogObject, content []byte) ([]byte, error) {
```

Comment:
The "Z" in the name isn't informative.
How about naming this function VerifyProtoSigningCertChain or something which captures that it is protobuf decode plus VerifySigningCertChain?